content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -753,6 +754,129 @@ Prisma CLI Version : 4.5.0
+| ---------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
+| `--schema`       | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
+| `--help` / `--h` | No       | Displays the help message                                                                                                                           |                                             |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+<CodeWithResult>
+
+<cmd>
+
+```terminal
+prisma debug
+```
+
+</cmd>
+
+<cmdResult>
+
+```text no-copy
+-- Prisma schema --
+Path: /prisma/schema.prisma
+
+-- Local cache directory for engines files --
+Path: /.cache/prisma
+
+-- Environment variables --
+When not set, the line is dimmed and no value is displayed.
+When set, the line is bold and the value is inside the `` backticks.
+
+For general debugging
+ - CI:
+ - DEBUG:
+ - NODE_ENV:
+ - RUST_LOG:
+ - RUST_BACKTRACE:
+ - NO_COLOR:
+ - TERM: `xterm-256color`
+ - NODE_TLS_REJECT_UNAUTHORIZED:
+ - NO_PROXY:
+ - http_proxy:
+ - HTTP_PROXY:
+ - https_proxy:
+ - HTTPS_PROXY:
+
+For more information about Prisma environment variables:
+See https://www.prisma.io/docs/reference/api-reference/environment-variables-reference
+
+For hiding messages
+ - PRISMA_DISABLE_WARNINGS:
+ - PRISMA_HIDE_PREVIEW_FLAG_WARNINGS:
+ - PRISMA_HIDE_UPDATE_MESSAGE:
+
+For downloading engines
+ - PRISMA_ENGINES_MIRROR:
+ - PRISMA_BINARIES_MIRROR (deprecated):
+ - PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING:
+ - BINARY_DOWNLOAD_VERSION:
+
+For configuring the Query Engine Type
+ - PRISMA_CLI_QUERY_ENGINE_TYPE:
+ - PRISMA_CLIENT_ENGINE_TYPE:
+
+For custom engines
+ - PRISMA_QUERY_ENGINE_BINARY:
+ - PRISMA_QUERY_ENGINE_LIBRARY:
+ - PRISMA_SCHEMA_ENGINE_BINARY:
+ - PRISMA_MIGRATION_ENGINE_BINARY:
+
+For the "postinstall" npm hook
+ - PRISMA_GENERATE_SKIP_AUTOINSTALL:
+ - PRISMA_SKIP_POSTINSTALL_GENERATE:
+ - PRISMA_GENERATE_IN_POSTINSTALL:
+
+For "prisma generate"
+ - PRISMA_GENERATE_DATAPROXY:
+ - PRISMA_GENERATE_NO_ENGINE:
+
+For Prisma Client
+ - PRISMA_SHOW_ALL_TRACES:
+ - PRISMA_CLIENT_NO_RETRY (Binary engine only):
+
+For Prisma Migrate
+ - PRISMA_SCHEMA_DISABLE_ADVISORY_LOCK:
+ - PRISMA_MIGRATE_SKIP_GENERATE:
+ - PRISMA_MIGRATE_SKIP_SEED:
+
+For Prisma Studio
+ - BROWSER:
+
+-- Terminal is interactive? --
+true
+
+-- CI detected? --
+false
+```
+
+</cmdResult>
+
+</CodeWithResult>
+
+<!-- prettier-ignore-end -->
+
+If you're using an older version of Prisma, you can use this command by running:
+
+```terminal
+prisma@latest debug
```

Comment:
```suggestion
npx prisma@latest debug
```

Reply:
Generally, we avoid prefixing the pkg manager in the Reference docs. 😄